pkg/resources/cloudcontroller/kubevirt.go

```diff
@@ -87,7 +87,7 @@ func kubevirtDeploymentReconciler(data *resources.TemplateData) reconciling.Name
 			dep.Spec.Template.Spec.Containers = []corev1.Container{
 				{
 					Name:         ccmContainerName,
-					Image:        registry.Must(data.RewriteImage(resources.RegistryQuay + "/kubermatic/kubevirt-cloud-controller-manager:" + KubeVirtCCMTag)),
+					Image:        registry.Must(data.RewriteImage(resources.RegistryQuay + "/kubevirt/kubevirt-cloud-controller-manager:" + KubeVirtCCMTag)),
```

Comment:
Do we want to push our own image?

Reply:
Honestly I am not sure. It took me ages to know if the image that we have in Quay is our image or the upstream image. Because we have the same fork in our org and the same version 😂